pulp_container/app/tasks/builder.py

```diff
@@ -111,12 +112,14 @@ def build_image_from_containerfile(
 
     """
     containerfile = Artifact.objects.get(pk=containerfile_pk)
+    containerfile.touch()
```

Comment:
i think, here, the line is not needed as well since we touched it in the viewset

Reply:
I had the same thougt. But then i decided to go ahead anyway, because there could be a considerate a amount of time between the request and the task, and it would be just so inconvenient if the artifact vanished between here and being used after reading all the other artifacts from cloud storage.

Reply:
with this logic then we should double that use of touch in viewset and inside of the task..
the default is set to 1day, i'd still insist on not doubling this.